pom.xml

```diff
@@ -44,11 +44,11 @@
   <distributionManagement>
     <snapshotRepository>
       <id>sonatype-nexus-snapshots</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      <url>https://google.oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
Did we want the snapshots to also go to the new endpoint?

Reply:
Yes - Joel recommended that we stay consistent in the last email. I will give this a try also.